docs/standard/threading/threading-objects-and-features.md

```diff
@@ -16,16 +16,16 @@ Along with the <xref:System.Threading.Thread?displayProperty=nameWithType> class
+|[Timers](timers.md)|Describes .NET timers that can be used in a multithreaded environment.|
+|[Overview of synchronization primitives](overview-of-synchronization-primitives.md)|Describes types that can be used to synchronize access to a shared resource or control thread interaction.|
 |[EventWaitHandle, AutoResetEvent, CountdownEvent, ManualResetEvent](eventwaithandle-autoresetevent-countdownevent-manualresetevent.md)|Describes managed event wait handles, which are used to synchronize thread activities by signaling and waiting for signals.|
-|[Mutexes](mutexes.md)|Describes how to use a <xref:System.Threading.Mutex?displayProperty=nameWithType> to synchronize access to an object or to build your own synchronization mechanisms.|
+|[Mutexes](mutexes.md)|Describes <xref:System.Threading.Mutex?displayProperty=nameWithType>, which grants an exclusive access to a shared resource.|
 |[Interlocked operations](interlocked-operations.md)|Describes the <xref:System.Threading.Interlocked?displayProperty=nameWithType> class, which provides atomic operations for variables that are shared by multiple threads.|
-|[Reader-Writer Locks](reader-writer-locks.md)|Describes the <xref:System.Threading.ReaderWriterLockSlim?displayProperty=nameWithType> class, which provides single-writer/multiple-reader semantics.|
-|[Semaphore and SemaphoreSlim](semaphore-and-semaphoreslim.md)|Describes the <xref:System.Threading.Semaphore?displayProperty=nameWithType> class and explains how to use it to control access to limited resources.|
+|[Reader-Writer Locks](reader-writer-locks.md)|Describes the <xref:System.Threading.ReaderWriterLockSlim?displayProperty=nameWithType> class, which provides single-writer/multiple-reader access to a shared resource.|
+|[Semaphore and SemaphoreSlim](semaphore-and-semaphoreslim.md)|Describes the <xref:System.Threading.Semaphore?displayProperty=nameWithType> class, which limits number of threads that can access a shared resource concurrently.|
 |[Barrier](barrier.md)|Describes the <xref:System.Threading.Barrier?displayProperty=nameWithType> class that implements the barrier pattern for coordination of threads in phased operations.|
-|[SpinLock](spinlock.md)|Describes the <xref:System.Threading.SpinLock?displayProperty=nameWithType> class, which is a lightweight alternative to the <xref:System.Threading.Monitor?displayProperty=nameWithType> class for certain low-level scenarios.|
-|[SpinWait](spinwait.md)|Describes the <xref:System.Threading.SpinWait?displayProperty=nameWithType> class, which is a low-level synchronization primitive that performs busy spinning prior to initiating a kernel-based wait.|
+|[SpinLock](spinlock.md)|Describes the <xref:System.Threading.SpinLock?displayProperty=nameWithType> struct, which grants an exclusive access to a shared resource with spin-based waiting.|
```

Comment:
I'd keep the original description - lightweight alternative and low-level are essential for spinlocks. Otherwise they wouldn't exist.